aws-codepipeline-transition-disabler/.gitignore

```diff
@@ -0,0 +1,5 @@
+/target/
```

Comment:
What's the point of adding .gitkeep files for these folders? They're not required.